### PR TITLE
Refactor kj

### LINKED ISSUE #39: Format CardDatabase page
- Decide how the links will be shown.

- small card pictures with titles?

### PR DESCRIPTION
**Pull request by kathrynljackson**
## Description

The commits included in this branch complete the basic planned formatting of the site. 

- Add `Journal.scss`
- Add Material Icons dependency
- Add reverse icon to reversed cards
- Make reversed cards image upside down on `CardPage`
- Add transition-duration for desktop styling
- Make `NavBar` responsive
- Format individual `CardPage`, is responsive
- Format buttons
- Make `Journal` page Dynamic
- Remove unused imports
- Reorder HTML elements for site uniformity

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

## Issues

- Closes #39

### CODE REVIEW

File: src/CardDatabase/CardDatabase.scss [hunk 1]
```diff
@@ -1,15 +1,28 @@
 .cards {
+    text-align: left;
+    padding: 4%;
+    font-family: Karla, sans serif;
```
Comment (CalebCyphers) [inline src/CardDatabase/CardDatabase.scss:4]: Cool font

File: src/CardPage/CardPage.js [hunk 2]
```diff
@@ -24,19 +21,33 @@ class CardPage extends Component {
...
-                    <img src={tarot.image} />
-                    <p>{tarot.theme}</p>
-                    <p>{tarot.description}</p>
-                    <Link to='/database'>Back</Link>
+                    <p className='tarot-numeral'>{tarot.numeral}</p>
+                    <p className='tarot-title'>{tarot.title}</p>
+                    <img className='tarot-image' src={tarot.image} alt={tarot.title}/>
+                    <p className='tarot-theme'>{tarot.theme}</p>
+                    <p className='tarot-description'>{tarot.description}</p>
+                    <Link className='tarot-back' to='/database'>Back</Link>
```
Comment (CalebCyphers) [inline src/CardPage/CardPage.js:33]: Doing the good work I see

File: src/DailyReading/DailyReading.js [hunk 3]
```diff
@@ -1,74 +1,31 @@
...
-          <Col>
-            <p>{props.card.description}</p>
-          </Col>
-        </Row>
+      <div className='individual-card'>
+        <p className='daily-card-numeral'>{props.card.numeral}</p>
+        <p className='daily-card-title'>{props.card.title}</p>
+        <img className='daily-card-image' src={props.card.image} alt={props.card.title} />
+        <p className='daily-card-theme'>{props.card.theme}</p>
+        <p className='daily-card-description'>{props.card.description}</p>
```
Comment (CalebCyphers) [inline src/DailyReading/DailyReading.js:15]: This is looking clean

File: src/NavBar/NavBar.scss [hunk 4]
```diff
@@ -1,32 +1,57 @@
...
+  margin-bottom: 3%;
+  padding: none !important;
+  border-bottom: 3px solid white;
+  border-top: 3px solid white;
+
+
+
+    
+
+    a:hover {
```
Comment (CalebCyphers) [inline src/NavBar/NavBar.scss:31]: love the responsiveness here

**Review by CalebCyphers — APPROVED** [review]

**Review by chadrick-d-dev — APPROVED** [review]
Great work cleaning things up!

**Pull request merged**